packages/amp/src/__tests__/processors.test.tsx

```diff
@@ -115,7 +171,6 @@ test("Validate amp-audio", async () => {
   );
   expect(container.firstChild).toMatchInlineSnapshot(`
     <amp-audio
-      controls="true"
```

Comment:
Previously we always passed a `controls` prop to `amp-audio` but we should not do that. The content should set that attribute.